types/vue2-datepicker/index.d.ts

```diff
@@ -1,16 +1,15 @@
-// Type definitions for vue2-datepicker 2.0
+// Type definitions for vue2-datepicker 2.12
 // Project: https://github.com/mengxiong10/vue2-datepicker
 // Definitions by: ChristianStornowski <https://github.com/ChristianStornowski>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 3.5
```

Comment:
Do you need to require that anyone using these types be on TypeScript 3.5 or later?